packages/inventory-compliance/src/SystemRulesTable.test.js

```diff
@@ -105,4 +105,19 @@ describe('SystemRulesTable component', () => {
         expect(wrapper.state('currentRows')[2].cells[TITLE_COLUMN].original).
         toEqual('Use direct-lvm with the Device Mapper Storage Driver');
     });
+
+    it('should render filtered rows with the right parent fields', async () => {
+        const wrapper = shallow(
+            <SystemRulesTable
+                profileRules={ profileRules }
+                loading={ false }
+                system={ system }
+            />
+        );
+        const instance = wrapper.instance();
+        await instance.setInitialCurrentRows();
+        await instance.updateFilter(false, [ 'high', 'medium' ], []);
+        expect(wrapper.state('currentRows').length / 2).toEqual(1);
+        expect(wrapper.state('currentRows')[1].parent).toEqual(0);
```

Comment:
Sometimes the behavior appears correct for one or two rows, but then another row will be messed up. Maybe we could go ahead and check that this is correct in a loop for `profileRules.length`

Reply:
There are only 2 rules in the sample request so I'll have to change that

Reply:
Yeah, I noticed that. :/ You could just set `profileRules = profileRules.concat(profileRules, profileRules, profileRules, ...)`